edx_exams/apps/lti/utils.py

```diff
@@ -5,7 +5,4 @@
 
 
 def get_lti_root():
-    if hasattr(settings, 'LTI_ROOT_URL_OVERRIDE'):
```

Comment:
we won't ever need a separate url for this anymore based on this setup: https://2u-internal.atlassian.net/wiki/spaces/PT/pages/256737327/Local+Development+LTI+Configuration